prism_strands/CMakeLists.txt

```diff
@@ -0,0 +1,250 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(prism_strands)
+
+
+
+## Compile as C++11, supported in ROS Kinetic and newer
+# add_compile_options(-std=c++11)
+
+## Find catkin macros and libraries
+## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
+## is used, also find other catkin packages
+find_package(catkin REQUIRED COMPONENTS
+)
+
+## System dependencies are found with CMake's conventions
+# find_package(Boost REQUIRED COMPONENTS system)
+
+include(ExternalProject)
+
+
+catkin_package(
+	#LIBRARIES mongoclient
+	#INCLUDE_DIRS include
+)
+
+ExternalProject_add(prism
+	URL http://www.prismmodelchecker.org/dl/prism-4.3.1-src.tar.gz
```

Comment:
is it possible to download from my github instead? https://github.com/bfalacerda/prism-svn/tree/robots

We're using our own branch of prism